apps/judicial-system/web/src/routes/Court/Indictments/Subpoena/Subpoena.tsx

```diff
@@ -254,7 +264,7 @@ const Subpoena: FC = () => {
           }}
           primaryButtonText={formatMessage(strings.modalPrimaryButtonText)}
           secondaryButtonText={formatMessage(strings.modalSecondaryButtonText)}
-          isPrimaryButtonLoading={false}
+          isPrimaryButtonLoading={isCreatingSubpoena}
```

Comment:
Note that there already exists a flag that tracks csae updates. Please consider using `isUpdatingCase` from `useCase` instead of defing your own.

Reply:
Discussed IRL, not doing